### PR TITLE
Feature/rcspf 129399 setup release process

### PR DESCRIPTION
**Pull request by ruadrian**

### CODE REVIEW

File: .github/actions/validate-release-version/index.js [hunk 1]
```diff
@@ -0,0 +1,24 @@
+const core = require('@actions/core');
+
+try {
+    // Validate input release version:
+    const releaseVersionInput = core.getInput('release-version');
+    const releaseVersionPattern = core.getInput('release-version-pattern');
+    console.log(`Validating release version ${releaseVersionInput} against pattern ${releaseVersionPattern}`);
+    const result = releaseVersionInput.match(new RegExp(`^${releaseVersionPattern}$`));
+    if(result == null) {
+        core.setFailed('Invalid release preparation tag, aborting release!');
```
Comment (ruadrian) [inline .github/actions/validate-release-version/index.js:10]: correct message

**Comment by ruadrian** [comment]
Add info about where to find gpg public key to readme?!

File: .github/scripts/release.sh [hunk 2]
```diff
@@ -0,0 +1,81 @@
...
+echo "Step $i: Push commits and release tag" ; ((++i))
+git_user=`git config --get user.name`
+git_email=`git config --get user.email`
+if [[ -z $git_user || -z $git_email ]] ; then
+  echo "No git user.name or user.email found!"
+  echo "Setting git user.name to github-actions and user.email to github-actions@github.com"
+  git config user.name github-actions
+  git config user.email github-actions@github.com  
+fi
+echo "git user.name: $git_user"
```
Comment (ruadrian) [inline .github/scripts/release.sh:48]: fetch user and email again after setting

File: ch.sbb.scion.rcp.workbench.test/META-INF/MANIFEST.MF [hunk 3]
```diff
@@ -1,11 +1,11 @@
 Manifest-Version: 1.0
 Bundle-ManifestVersion: 2
 Bundle-Name: SCION Workbench Test Fragment
-Bundle-SymbolicName: rcp.workbench.test;singleton:=true
+Bundle-SymbolicName: rcp-workbench-test;singleton:=true
```
Comment (iachelini) [inline ch.sbb.scion.rcp.workbench.test/META-INF/MANIFEST.MF:4]: I think the .project file needs to be changed too, right?

**Review by iachelini — DISMISSED** [review]

**Review by iachelini — APPROVED** [review]

**Pull request merged**